File: WC1/square2.c
#include <stdio.h>

int main() {
    int size;
    
    scanf("%d", &size);
    
    for ( int row = 0, counter = 1; row < size; row++ ) {
        for ( int column = 1; column < size; column++ ) {
            printf("%d ", counter);
            counter += 1;
        }
        printf("%d\n", counter);
        counter += 1;
    }
    
    return 0;
}

File: WC4/PHP/square2.php
<?php
    $size = fgets(STDIN);
    
    for ( $row = 0, $counter = 1; $row < $size; $row++, $counter++ ) {
        for ( $column = 1; $column < $size; $column++, $counter++ ) {
            echo $counter . ' ';
        }
        echo $counter . PHP_EOL;
    }
    
?>